.pre-commit-config.yaml

```diff
@@ -0,0 +1,60 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-added-large-files
+      - id: check-json
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: check-toml
+      - id: check-yaml
+        args: [--unsafe]
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.790
+    hooks:
+      - id: mypy
+  - repo: https://github.com/ambv/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        name: Run black formatter
+        language_version: python3.8
+  - repo: https://gitlab.com/pycqa/flake8
```

Comment:
Default line length in [pep8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) is 79. [Default of black](https://github.com/psf/black) is 88.

Reply:
I like black! :-). The 79 line length in pep8 is a relic from when the screens weren't wider than 80 chars. But I do not care so much. As long as it's not to complicated to contribute because you must know every nitty gritty piece of the CI/CD workflow I am alright.

Reply:
Agreed. I would suggest to remove the `mypy`, that highers the bar for contributing (a lot). 

`black` makes the code of multiple contributor more consistent in style, it requires the contributor to run black locally if their changes do not adhere to black.